modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/objectLayoutContext.tsx

```diff
@@ -235,10 +235,7 @@ const layoutReducer = (state: TState, action: TAction) => {
 				(box) => box.type !== 'regular'
 			);
 
-			if (
-				type === 'regular' ||
-				(type === 'categorization' && frameworkIndex >= 0)
-			) {
```

Comment:
Hey @henriquelimadotcom, removing this condition we will create another bug which is the `categorization block` must be the last block in a tab, if we remove the `type === 'regular'` from the condition we could add a block below the `categorization block`. So to avoid this we need to improve this condition, for that first we can get the information if the `objectLayoutBoxes` already has a `categorization block` or not, we can do this using the `find()` function:

```tsx
const hasCategorizationBox = !!objectLayoutBoxes.find(
    layoutBox => layoutBox.type === 'categorization'
);
```

After that we can use this information to improve the condition by checking if the type is `regular` and has the `categorization box`:

```tsx
if (
    (type === 'regular' && hasCategorizationBox) ||
    (type === 'categorization' && frameworkIndex >= 0)
) {
    objectLayoutBoxes.splice(frameworkIndex, 0, newBox);
}
```